Added warning making sure to setup/init logging before resolving logger (#4341)
Particular/docs.particular.net: nservicebus/logging/usage.md

snippet: UsingLogging

> WARNING: Make sure that logging is correctly setup/initialized before resolving a logger instance. Not doing this can result a logger using an incorrect configuration

Reviewer: ```suggestion
WARNING: Make sure that logging is correctly initialized before resolving the `ILog` instance. Not doing so can result in a logger using an incorrect configuration
```

Author: @kbaley Its a common problem, but that doesn't mean we should't warn them. Its based on support and this isn't the first time I had this issue.I like to see it added.
